src/logic/safe/transactions/offchainSigner/index.ts

```diff
@@ -13,16 +15,31 @@ const SIGNERS = {
   ETH_SIGN: ethSigner,
 }
 
+export const SAFE_VERSION_FOR_OFF_CHAIN_SIGNATURES = '>=1.0.0'
+
 // hardware wallets support eth_sign only
-const getSignersByWallet = (isHW) =>
-  isHW ? [SIGNERS.ETH_SIGN] : [SIGNERS.EIP712_V3, SIGNERS.EIP712_V4, SIGNERS.EIP712, SIGNERS.ETH_SIGN]
+// eth_sign is only supported by safes >= 1.1.0
+const getSignersByWallet = (isHW: boolean, safeVersion: string) => {
```

Comment:
I think this could be renamed to `getSupportedSigners` because now it checks more than just a wallet

Reply:
done!